.gitignore

```diff
@@ -37,6 +37,7 @@ sphinx-output
 .venv*
 .cache
 .commands
+.python-version
```

Comment:
może jednak zostawmy .python-version w localu to dzięki temu zaczniemy wymuszać wersję. Wiem że wielu developerów tak zaczęło robić